Source/Core/DolphinQt/Debugger/MemoryViewWidget.h

```diff
@@ -54,24 +71,38 @@ class MemoryViewWidget final : public QWidget
     WriteOnly
   };
 
+  enum class UpdateType
+  {
+    Full,
+    Addresses,
+    Values,
+    Auto,
+  };
+
   explicit MemoryViewWidget(Core::System& system, QWidget* parent = nullptr);
 
   void CreateTable();
+  void UpdateDisbatcher(UpdateType type = UpdateType::Addresses);
```

Comment:
Maybe intentional; or did you mean
Dis**p**atcher rather than Dis**b**atcher?

Reply:
Oh, lol. Thanks, yes.